docs/kbs_attestation_protocol.md

```diff
@@ -1,220 +1,418 @@
-and then send from the same KBC multiple Requests. Having to do the whole attestation dance would 
-be time consuming and have a latency impact on the guest.
-
-In order to ensure the ease of use and security completeness of KBS, we will use HTTPS 
-as the default transmission method to carry the application layer semantics designed in this 
-document. This is because HTTPS provides KBC with a means to authenticate KBS identity, which 
-effectively avoids malicious attackers from hijacking KBS address to impersonate KBS and deceive KBC.
-In order to achieve this, the public key of KBS needs to be transmitted to KBC through an effective way, 
-and the specific way of public key distribution is not within the scope of this document.
-In addition, it should be noted that the confidentiality protection provided by HTTPS alone is not enough 
-to meet the security requirements of KBS protocol, therefore, as mentioned above, the key generated 
-by HW-TEE needs to be used to encrypt and protect the confidential data.
-
-# RCAR semantics
-
-The semantics of attestation defined by KBS is a simple and extensible four-step model, 
-which uses JSON structure to organize information. As follows:
-
-1. **Request**: The KBC sends to KBS, calls the service API provided by KBS to request resources.
-2. **Challenge**: After receiving the request, KBS returns a challenge to KBC and asks KBC to 
-send evidence to prove that its environment (HW-TEE) is safe and reliable.
-3. **Attestation**: KBC sends the evidence and the HW-TEE generated, ephemeral public key to the KBS.
-4. **Response**: The `Response` payload includes the resource data requested when sending the initial `Request` to
-the specific KBS resource endpoint. The resource data is protected by the ephemeral public key
-passed via the previously sent Attestation payload. Within the valid time of the token, KBC can directly 
-request resources or services from KBS by virtue of the token without going through step 2 and step 3.
-
-## Request
+The purpose of the attestation between KBS and KBC is to confirm whether **the
+platform where the KBC is located is in the expected security state**, i.e. if
+it runs in a real and harmless HW-TEE, and the software stack is measured and
+verifiable.
+
+In this document, HW-TEE attestation process is the semantics of the application
+layer, which is defined as a simple, universal and extensible
+"Request-Challenge-Attestation-Response" (RACR) protocol. The temporary
+asymmetric key generated by HW-TEE is used to encrypt the response output
+payload, and the standard [HTTP Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies)
+mechanism is used to avoid the performance problems caused by multiple
+attestation: it's fairly typical to request for several resources, and then send
+from the same KBC multiple Requests. Having to do the whole attestation dance
```

Comment:
This might be slightly clearer?:

```suggestion
attestation: it's fairly typical for the KBC to require several resources, resulting in multiple Requests to the KBS. Having to do the whole attestation dance
```

Reply:
Fixed, thanks.